neuralop/models/padding.py

```diff
@@ -54,23 +58,37 @@ def pad(self, x):
             
             print(f'Padding inputs of {resolution=} with {padding=}, {self.padding_mode}')
 
+            output_pad = padding
+
+            for scale_factor in self.output_scale_factor:
+                if isinstance(scale_factor, (float, int)):
+                    scale_factor = [scale_factor]*len(resolution)
+                output_pad = [int(round(i*j)) for (i,j) in zip(scale_factor,output_pad)]
```

Comment:
Why the for loop? You're just overwriting output_pad at each iteration here. 
If scale_factor is a list shouldn't we be using one per mode?
We should also document the parameter